interp/constrexpr_ops.ml

```diff
@@ -411,13 +410,24 @@ let fold_constr_expr_with_binders g f n acc = CAst.with_val (function
       acc
   )
 
+let rec free_vars_of_constr_expr_gen bdvars l = function
```

Comment:
I know it was already there in the previous code but calling `l` something of type `Id.Set.t` feels... counterintuitive

Reply:
I renamed into `ids`.